snippets/2_tricky_strings.py

```diff
@@ -0,0 +1,31 @@
+# 1
+assert id("some_string") == id("some" + "_" + "string")
+assert id("some_string") == id("some_string")
+
+# 2
+a = "wtf"
+b = "wtf"
+assert a is b
+
+a = "wtf!"
+b = "wtf!"
+# True because it is invoked in script. Might be False in python shell or ipython
+assert a is b
+
+# 3
+a, b = "wtf!", "wtf!"
+assert a is b
+
+a = "wtf!"; b = "wtf!"  # noqa: E702 - multiline statement
```

Comment:
instead of adding noqa here (which might confuse beginners), let's ignore this rule in linter settings

Reply:
Sure, it is better to choose and setup linter first and then add `noqa` or project level ignores